Sources/DittoDiskUsage/DiskUsageViewModel.swift

```diff
@@ -67,8 +67,8 @@ public class DiskUsageViewModel: ObservableObject {
     @Published var diskUsage: DiskUsageState?
     var cancellable: Cancellable?
 
-    /// The size over which disk usage is considered unhealthy when used as a `HealthMetric` with the heartbeat tool (this only considers `ditto_store` and `ditto_replication`). Defaults to 2GB
-    var unhealthySizeInBytes: Int = DittoDiskUsageConstants.twoGigabytesInBytes
+    /// The size over which disk usage is considered unhealthy when used as a `HealthMetric` with the heartbeat tool (this only considers `ditto_store` and `ditto_replication`). Defaults to 500MB
+    public var unhealthySizeInBytes: Int = DittoDiskUsageConstants.fiveHundredMegabytesInBytes
```

Comment:
Making this public allows anyone to override in case they would like to see lower/higher limits?

Reply:
Yes, we think this is a good baseline but its possible for different customers with different usecases to run into this at lower or higher numbers based on document size.

It was a miss to not have it public the first time around.